hugo/layouts/partials/header.html

```diff
@@ -1,30 +1,72 @@
-<nav class="fullWidth topbar {{ .Params.topbar }}" id="topBar">
+<nav class="topbar" id="topbar">
     <div>
         <a href="{{ .Site.BaseURL }}" alt="source{d}">
-            <div class="logo"></div>
+            <div class="logo"><img src="{{ .Site.BaseURL }}/img/logos/source-logo-white.svg" alt="source{d}" /></div>
         </a>
-        <div class="menu" id="menu">
+        <div class="topbar__menu" id="menu">
             <ul>
                 <li {{ if eq .URL "/" }}class="active"{{ end }} title="homepage">
-                    <a href="{{ .Site.BaseURL }}">home</a>
+                    <a href="{{ .Site.BaseURL }}" title="Home">Home</a>
                 </li>
                 <li {{ if eq .URL "projects" }}class="active"{{ end }}>
-                    <a href="{{ .Site.BaseURL }}/projects" title="Projects">projects</a>
+                    <a href="{{ .Site.BaseURL }}/projects" class="topbar__menu--projects" id="submenu-toggle" title="Projects">
+                        <span>Projects</span>
+                        {{ partial "svg" "go-arrow-nocircle" }}
+                    </a>
+
+                    <ul class="topbar__submenu" id="submenu">
+                        <li>
```

Comment:
It's required that the category of a project is "active" when it is being shown a project of that category.
Then, it will be needed to identify each category link, and define css rules to let this behavior happen